todo/src/reducers/index.js

```diff
@@ -0,0 +1,14 @@
+import { ADD_TODO, FETCH_TODOS, DELETE_TODO } from '../actions';
+
+export const todoReducer = (state = [], action) => {
+    switch (action.type) {
+        case ADD_TODO:
+            return state.concat(action.payload);
+        case FETCH_TODOS:
+            return state.concat(action.payload);
+        case DELETE_TODO:
+            return state.filter(todo => todo !== action.payload);
```

Comment:
Here's a stretch problem, if you're interested: If you handle your deletions this way, and you have two tasks that have the same text, then clicking the button to delete one will remove both. This may or may not be desired behavior. How would you avoid it?